File: twosum.py
from typing import List


class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        twosum_dict = dict()
        for idx, value in enumerate(nums):
            try:
                return [twosum_dict[value], idx]
            except:
                twosum_dict[target - value] = idx

Solution().twoSum([2,7,11,4] , 15)
try:
    print('help')
except:
    print('aw man')

File: twoSum.c
int* twoSum(int* nums, int numsSize, int target, int* returnSize)
{
    *returnSize = 2;
    int* ret =  malloc(sizeof(int) * 2);

    for( int i = 0; i < numsSize; i++)
    {
        for (int j = i + 1; j < numsSize; j++)
        {
            if( nums[i] + nums[j] == target)
            {
               ret[0] = i; ret[1] = j;
               return ret;
            }
        }
    }
    return ret;
}